apps/console/src/features/users/components/bulk-import-response-list.tsx

```diff
@@ -0,0 +1,391 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AlertTitle } from "@mui/material";
+import Alert from "@oxygen-ui/react/Alert";
+import Typography from "@oxygen-ui/react/Typography";
+import {
+    DataTable,
+    EmptyPlaceholder,
+    LinkButton,
+    ListLayout,
+    TableColumnInterface
+} from "@wso2is/react-components";
+import React, { ReactElement, ReactNode, useEffect, useState } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import {
+    Dropdown,
+    DropdownItemProps,
+    DropdownProps,
+    Grid,
+    Header,
+    Label
+} from "semantic-ui-react";
+import {
+    AdvancedSearchWithBasicFilters, UIConstants, getEmptyPlaceholderIllustrations } from "../../core";
+import { BulkUserImportStatus } from "../constants";
+import { BulkResponseSummary, BulkUserImportOperationResponse } from "../models";
+
+interface BulkImportResponseListProps {
+    responseList: BulkUserImportOperationResponse[];
+    isLoading?: boolean;
+    ["data-componentid"]?: string;
+    bulkResponseSummary?: BulkResponseSummary
+    hasError: boolean;
+}
+
+const ALL_STATUS: string = "ALL";
+
+type FilterStatus = BulkUserImportStatus | typeof ALL_STATUS;
```

Comment:
Shall we add "ALL" to the BulkUserImportStatus enum and remove ALL_STATUS variable here?

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/6597.